content/using/install.md

```diff
@@ -1,42 +1,38 @@
 +++
-title = "Install + Setup"
+title = "Urbit Quickstart Guide"
```

Comment:
My personal opinion, btw, is that the word "Install" is sort of important, at least it's the first thing I look for when I scope out a project...

Reply:
I don't feel strongly, so I'm happy to revert.